docs/self-managed/platform-deployment/ingress-setup.md

```diff
@@ -0,0 +1,164 @@
+:::
+
+## Preparation
+
+- An Ingress controller should be deployed in advance. The examples below use `Nginx` Ingress controller, but any Ingress controller could be used by setting `ingress.className`.
+- TLS configuration is not handled in the examples because it varies between different workflows. It could be configured directly using `ingress.tls` options or via an external tool like [Cert-Manager](https://github.com/cert-manager/cert-manager) using `ingress.annotations`. For more details, check available [configuration options](https://github.com/camunda/camunda-platform-helm/tree/main/charts/camunda-platform#configuration).
+
+## Combined Ingress setup
+
+In this setup, a single Ingress/domain is used to access Camunda Platform web applications, and another for Zeebe Gateway. By default, all web applications use `/` as a base, so we just need to set the context path, Ingress configuration, and authentication redirect URLs.
+
+```yaml
+# Chart values for the Camunda Platform 8 Helm chart in combined Ingress setup.
+
+# This file deliberately contains only the values that differ from the defaults.
+# For changes and documentation, use your favorite diff tool to compare it with:
+# https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform
+
+# IMPORTANT: Make sure to change "camunda.example.com" to your domain.
+
+global:
+  ingress:
+    enabled: true
+    className: nginx
+    host: "camunda.example.com"
+  identity:
+    auth:
+      publicIssuerUrl: "https://camunda.example.com/auth/realms/camunda-platform"
+      operate:
+        redirectUrl: "https://camunda.example.com/operate"
+      tasklist:
+        redirectUrl: "https://camunda.example.com/tasklist"
+      optimize:
+        redirectUrl: "https://camunda.example.com/optimize"
+
+identity:
+  contextPath: "/identity"
+  fullURL: "https://camunda.example.com/identity"
+
+operate:
+  contextPath: "/operate"
+
+optimize:
+  contextPath: "/optimize"
+
+tasklist:
+  contextPath: "/tasklist"
+
+zeebe-gateway:
+  ingress:
+    enabled: true
+    className: nginx
+    host: "zeebe.camunda.example.com"
+```
+
+Using the custom values file, [deploy Camunda Platform as usual](../deployment.md):
+
+```shell
+helm install demo camunda/camunda-platform -f values-combined-ingress.yaml
+```
+
+Once deployed, you can access the Camunda Platform components on:
+
+- **Web applications:** `https://camunda.example.com/[identity|operate|optimize|tasklist]`
+- **Keycloak authentication:** `https://camunda.example.com/auth`
+- **Zeebe Gateway:** `grpc://zeebe.camunda.example.com`
+
+## Separated Ingress setup
+
+In this setup, each Camunda Platform component has its own Ingress/domain. There is no need to set the context (unless there is a good reason for that) since `/` is used as a default base. Here, we just need to set the Ingress configuration and authentication redirect URLs.
```

Comment:
"(unless there is a good reason for that)" -- can you clarify here? Are there particular instances or scenarios where there might be a good reason? If there aren't any worth mentioning, I would remove this.

Reply:
yeah if the domain will be used for multiple resources.
but I'd say who does that will know what to do ... so I will remove it.